templates/components/molecules/icon-grid/icon-grid.twig

```diff
@@ -6,7 +6,7 @@
   {% if items is not empty %}
   {% set classes = "" %}
     {% for item in items %}
-      {% include "@decanter/components/molecules/media-block-row/media-block-row.twig" with item %}
+      {% include "../../molecules/media-block-row/media-block-row.twig" with item %}
```

Comment:
Where did the namespace go? This is important to keep if we plan to use the templates in other systems.